odtp/dashboard/utils/storage.py

```diff
@@ -26,7 +27,9 @@
 class ODTPFormValidationException(Exception):
     pass
 
-
+def reset_all () -> None: 
```

Comment:
This function is never called, but should be called on logout. Also please correct: `reset_all()`: best run `black` after you have finished work on a code file. It will fix the linting.